applications/charter/P4_Apps_WG_charter.mdk

```diff
@@ -0,0 +1,54 @@
+Title         : P4 Applications WG Charter
+
+[TITLE]
+
+# Working Group Name
+
+P4 Applications Working Group
+
+# Chairs
+
+- Jeongkeun Lee <jklee@barefootnetworks.com>
+- Mukesh Hira <mhira@vmware.com>
+
+# Mailing Lists
+
+- To subscribe, visit
+  [http://lists.p4.org/mailman/listinfo/p4-apps_lists.p4.org](http://lists.p4.org/mailman/listinfo/p4-apps_lists.p4.org)
+- To post a message to all list members, end an email to <p4-apps@lists.p4.org>
+
+# Description of the Working Group
+The P4 development environment is maturing. Developers have started to create
+new, innovative data-plane applications using P4. These applications will 
+change the way people design, build, and run not just their networks, but 
```

Comment:
I would phrase this as a positive stmt: and run their networks, as well as compute and storage infrastructure.